themes/bootstrap3/templates/RecordTab/similaritemscarousel.phtml

```diff
@@ -17,8 +17,7 @@
         <?php foreach ($similarRecords as $index => $data): ?>
           <div class="carousel-item">
             <a class="hover-overlay" href="<?=$this->recordLink()->getUrl($data)?>">
-              <?php $thumb = $this->record($data)->getThumbnail('large'); ?>
-              <img src="<?=$thumb ?>" title="<?=$data->getTitle() ?>"/>
+              <?=$this->record($data)->getCoverDetails('core', 'large')['html']?>
```

Comment:
This was not displaying correctly in the previous version; I think I've fixed it by simplifying the code, but please let me know if I've failed to account for anything.

Reply:
It's loooking good. Thank!